pkg/apis/admission/BUILD

```diff
@@ -0,0 +1,46 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
Are we sure this is advisable?  If you run `./hack/update-all.sh` as suggested in the [documentation](https://github.com/kubernetes/community/blob/master/contributors/devel/adding-an-APIGroup.md), this is created.  Also, for new people it would be hard to distinguish between the different _types_ of autogenerated code, like bazel vs. codecgen vs. protobuf vs. openapi/swagger.  Following the documentation has all of these resources being created.